autoload/bazel/workspace.vim

```diff
@@ -15,7 +15,7 @@
 function! bazel#workspace#GetBazelRootPath(path) abort
   let l:path = s:AbsoluteDirectoryPath(a:path)
   if empty(l:path)
-    throw maktaba#error#NotFound('No workspace found')
+    throw maktaba#error#InvalidArguments('Empty path')
```

Comment:
I think BadValue fits better, and `:h ERROR(InvalidArguments)` says to prefer BadValue if relevant.